### ISSUE TITLE
Ithenticate fails because of a key error related to Django's SafeString type

### ISSUE THREAD

**Issue by joemull** [BirkbeckCTP/janeway#3868]
**Stack trace**

The following error occurs when ithenticate is run:

```
KeyError: <class 'django.utils.safestring.SafeString'>
  File "xmlrpc/client.py", line 522, in __dump
    f = self.dispatch[type(value)]

TypeError: cannot marshal <class 'django.utils.safestring.SafeString'> objects
  File "django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "security/decorators.py", line 66, in wrapper
    return func(request, *args, **kwargs)
  File "security/decorators.py", line 276, in wrapper
    return func(request, *args, **kwargs)
  File "review/views.py", line 119, in unassigned_article
    id = ithenticate.send_to_ithenticate(article, file)
  File "utils/ithenticate.py", line 99, in send_to_ithenticate
    submission = server['server'].document.add(
  File "xmlrpc/client.py", line 1122, in __call__
    return self.__send(self.__name, args)
  File "xmlrpc/client.py", line 1461, in __request
    request = dumps(params, methodname, encoding=self.__encoding,
  File "xmlrpc/client.py", line 981, in dumps
    data = m.dumps(params)
  File "xmlrpc/client.py", line 514, in dumps
    dump(v, write)
  File "xmlrpc/client.py", line 536, in __dump
    f(self, value, write)
  File "xmlrpc/client.py", line 607, in dump_struct
    dump(v, write)
  File "xmlrpc/client.py", line 536, in __dump
    f(self, value, write)
  File "xmlrpc/client.py", line 589, in dump_array
    dump(v, write)
  File "xmlrpc/client.py", line 536, in __dump
    f(self, value, write)
  File "xmlrpc/client.py", line 607, in dump_struct
    dump(v, write)
  File "xmlrpc/client.py", line 532, in __dump
    raise TypeError("cannot marshal %s objects" % type(value))
```

This might have to do with BleachField marking the article title as safe.

**Janeway version**
63d32009f60bf815edb629815b26f3156f31d0e2

**Comment by ajrbyers** [comment]
@joemull I can confirm this is the issue. Happy to look into it tomorrow for RC-5.

**Comment by mauromsl** [comment]
closed by #3983
